package/Dockerfile.dapper-base

```
@@ -41,3 +41,11 @@ RUN dnf -y distrosync && \
     curl -Lo /usr/bin/kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/kind-linux-${ARCH}" && chmod a+x /usr/bin/kind && \
     GOFLAGS="" go get -v github.com/onsi/ginkgo/ginkgo && \
     GOFLAGS="" go get -v golang.org/x/tools/cmd/goimports
+
+# Copy CI deployment scripts into image to share with all submariner-io/* projects
+RUN mkdir -p /opt/kind-e2e
+COPY scripts/kind-e2e/e2e.sh /opt/kind-e2e/e2e.sh
+COPY scripts/kind-e2e/lib_operator_deploy_subm.sh /opt/kind-e2e/lib_operator_deploy_subm.sh
+COPY scripts/kind-e2e/lib_helm_deploy_subm.sh /opt/kind-e2e/lib_helm_deploy_subm.sh
+COPY scripts/kind-e2e/nginx-demo.yaml /opt/kind-e2e/nginx-demo.yaml
+COPY scripts/kind-e2e/netshoot.yaml /opt/kind-e2e/netshoot.yaml
```

To reduce the number of layers, this should be a single `COPY` command:

```suggestion
COPY scripts/kind-e2e/e2e.sh scripts/kind-e2e/lib_operator_deploy_subm.sh scripts/kind-e2e/lib_helm_deploy_subm.sh scripts/kind-e2e/nginx-demo.yaml scripts/kind-e2e/netshoot.yaml /opt/kind-e2e/
```

---

Fixed, thanks!